examples/device/device.cpp

```diff
@@ -77,6 +81,24 @@ struct CXX_QDMI_Operation_impl_d {
   std::string name;
 };
 
+struct CXX_QDMI_TelemetrySensor_impl_d {
+  std::string id;
+  std::string unit;
+  std::chrono::seconds sampling_rate;
+};
+
+struct CXX_QDMI_Device_TelemetrySensor_Query_impl_d {
+  std::chrono::time_point<std::chrono::system_clock> start_time;
+  std::chrono::time_point<std::chrono::system_clock> end_time;
+  size_t timeout{};
```

Comment:
Could that be seconds, i.e.
```suggestion
  [size_t](std::chrono::seconds) timeout{};
```
?

Reply:
```suggestion
  size_t timeout;
```
and `size_t` (same as `std::chrono::seconds`) does not need a `{}` initialiser.

Reply:
I believe it cannot. However, we can `std::chrono::seconds` instead of `size_t`, leading to additional casting in the `CXX_QDMI_device_telemetrysensor_query_wait`. If you suggest that it will improve the readability of the code, we can change it.

Reply:
✅ Done